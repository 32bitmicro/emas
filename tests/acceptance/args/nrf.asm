nrf
nrf 0
nrf 1
nrf 255
