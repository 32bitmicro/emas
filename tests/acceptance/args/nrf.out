@ 0x0000 : 0xe7ff  /  111 001 1 111 111 111  /  59391
@ 0x0001 : 0xe700  /  111 001 1 100 000 000  /  59136
@ 0x0002 : 0xe701  /  111 001 1 100 000 001  /  59137
@ 0x0003 : 0xe7ff  /  111 001 1 111 111 111  /  59391
